l1infotreesync/l1infotreesync.go

```diff
@@ -281,6 +294,12 @@ func (s *L1InfoTreeSync) GetFirstInfo() (*L1InfoTreeLeaf, error) {
 	}
 	return s.processor.GetFirstInfo()
 }
+func (s *L1InfoTreeSync) GetInfoLeafByRoot(root common.Hash) (*L1InfoTreeLeaf, error) {
```

Comment:
To be consistent with other function names that return the `L1InfoTreeLeaf``:

```suggestion
func (s *L1InfoTreeSync) GetInfoByRoot(root common.Hash) (*L1InfoTreeLeaf, error) {
```

Reply:
f131a1c3496af5da1e0efc4b952557970225fdd1